src/SIM/SIMbase.C

```diff
@@ -1582,14 +1582,21 @@ void SIMbase::getBoundaryNodes (int pcode, IntVec& glbNodes, Vec3Vec* XYZ) const
     if (abs(p->pindx) == pcode && (pch = this->getPatch(p->patch)))
       switch (pch->getNoParamDim() - abs(p->ldim)) {
       case 1: // The boundary is of one dimension lower than the patch
-        pch->getBoundaryNodes(abs(p->lindx),glbNodes);
+      {
+        IntVec glbNodes2;
+        pch->getBoundaryNodes(abs(p->lindx),glbNodes2);
+        for (const auto& it : glbNodes2)
+          if (std::find(glbNodes.begin(), glbNodes.end(), it) == glbNodes.end())
+            glbNodes.push_back(it);
+
```

Comment:
Hm, nå er jeg visst ikke helt med. Trodde at denne sjekkingen om unike noder var for å ta høyde for at noen noder kunne være brukt av flere basiser for mixed, og dermed kunne listes to ganger uten sjekken. Og da er det vel uavhengig av om det er FSI eller ei? Eller hvordan tar jeg feil her. Hvis det _ikke_ er det (dvs., som jeg mistenker, hver basis har sitt eget sett med noder), så burde ikke denne testen vært nødvendig i det hele tatt.